opensrp-anc/src/test/java/org/smartregister/anc/library/presenter/PreviousContactDetailsPresenterTest.java

```diff
@@ -96,7 +96,7 @@ public void testLoadPreviousContacts() throws ParseException, JSONException, IOE
         PowerMockito.mockStatic(AncLibrary.class);
         PowerMockito.mockStatic(TextUtils.class);
         PowerMockito.when(!TextUtils.isEmpty(null)).thenReturn(true);
-        PowerMockito.when(AncLibrary.getInstance()).thenReturn(AncLibrary);
+        PowerMockito.when(org.smartregister.anc.library.AncLibrary.getInstance()).thenReturn(AncLibrary);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'org.smartregister.anc.library.AncLibrary.getInstance' due to existing import 'org.smartregister.anc.library.AncLibrary'](https://app.codacy.com/manual/OpenSRP/opensrp-client-anc/pullRequest?prid=4809639)